### DISCUSSION TITLE
UndefVarError: `video` not defined

### DISCUSSION THREAD

**Discussion by deahhh** [GenieFramework/Genie.jl#648] [Q&A]
```Julia
Html.html(path"/page/videos.jl.html", videos=videos, layout=:app)
```
videos.jl.html
```html
<video src="#"></video>
```

Caused by lack of video funtion in Genie.Renderer.Html ?
How to avoid it?

Thanks!

**Comment by deahhh** [comment]
I solve it by add `:video` to `NORMAL_ELEMENTS` in Html.jl in line#23.

Would you update it to the repo?

**Comment by essenciary** [comment]
Yes, can you please submit a PR?
